src/lsg/patterns/view-switch/index.tsx

```diff
@@ -0,0 +1,80 @@
+import { colors } from '../colors';
+import { Size } from '../copy';
+import { Icon, IconName, IconProps, Size as IconSize } from '../icons';
+import * as React from 'react';
+import { getSpace, Size as SpaceSize } from '../space';
+import styled from 'styled-components';
+
+export interface ViewSwitchProps {
+	fontSize?: Size;
+	justify?: 'start' | 'center' | 'end' | 'stretch';
+	leftVisible: boolean;
+	onLeftClick: React.MouseEventHandler<SVGElement>;
```

Comment:
please name all event handler `handle...`. This way they are a lot easier to differentiate from the native react event handlers

Reply:
What I've so far understood is that the convention is:
1. onClick: for elements just receiving or passing the event, like in this case the `view-switch`
2. handleClick: for the component that is actually operating or acting on the click event.

What is your opinion about that?

Reply:
Currently we only use `on` when we use a pass something to a native react event prop.
This has the benefit that you can immediately see which event handlers are custom ones and could have a different response than you would assume.